src/NuGet.Core/NuGet.CommandLine.XPlat/XPlatUtility.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Net;
+using Microsoft.Dnx.Runtime.Common.CommandLine;
+using NuGet.Common;
+using NuGet.Logging;
+using NuGet.Protocol.Core.Types;
+
+namespace NuGet.CommandLine.XPlat
+{
+    public class XPlatUtility
+    {
+        internal const string HelpOption = "-h|--help";
+        internal const string VerbosityOption = "-v|--verbosity <verbosity>";
+        internal static readonly int MaxDegreesOfConcurrency = Environment.ProcessorCount;
```

Comment:
Typically you want anything that is processor bound to be x2 number of processors. It will probably be a good idea to expose an environment variable to control this if anything goes wrong in prod (too much or too little parallelism), otherwise we have no control, ok with making this an issue and not resolving at the moment